enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -50,11 +51,40 @@
   {% else %}
     {{ enterprise_customer|capfirst }}
   {% endif %}
-  &rsaquo; {% trans "Manage Learners" %}
+  &rsaquo;
+  {% if search_keyword %}
+    <a href="{% url 'admin:enterprise_manage_learners' enterprise_customer.uuid %}">
+    {% trans "Manage Learners" %}
+    </a>
+  {% else %}
+    {% trans "Manage Learners" %}
+  {% endif %}
+  {% if search_keyword %}
+    &rsaquo; {% trans "Search Term: " %}"{{ search_keyword }}"
+  {% endif %}
 </div>
 {% endblock %}
 
 {% block content %}
+<div id="content-main">
+<div class="module" id="changelist">
```

Comment:
Is there a technical reason why we are not indenting the elements under `<div id="content-main">`?